src/common/utils/model.ts

```diff
@@ -14,6 +14,7 @@ import moment from 'moment';
 import { assert } from '.';
 import { getSnapshotUrlOverride } from './snapshots';
 import { Region, County, State, getStateCode } from 'common/regions';
+import { fail } from 'assert';
```

Comment:
I think we actually want to import it from 'common/utils' which is our own implementation (which among other things plays better with TypeScript) instead of the node.js one.